src/graphql/triage.rs

```diff
@@ -4,6 +4,7 @@ mod policy;
 pub(super) mod response;
 
 use async_graphql::{Enum, ID, InputObject, Object};
+use attrievent::attribute as attr;
```

Comment:
This shouldn't be used here in this way, as there's no guarantee that the versions of `attrievent` used by `review-web` and `review-database` are the same. 

To resolve this dependency issue, `review-database` should re-export the necessary data types from `attrievent` to `review-web`. This would allow `review-web` to depend solely on `review-database`, without directly relying on `attrievent`.

That said, this change would require modifications in `review-database` and would significantly impact a series of pending PRs, making it extremely difficult to merge them smoothly.

Therefore, I will merge this as it is for now, but this issue should be properly addressed once the affected PRs have been merged.

(Cc: @sophie-cluml and @msk)

Reply:
@kimhanbeom, could you create corresponding issues?